matrix.py

```diff
@@ -82,10 +82,16 @@ def merge_config(args, config):
 def setup(config):
     """Sets up the Matrix client. Makes sure the (specified) room is joined.
     """
+    loginargs = {}
+    if 'token' in config:
+        loginargs['user_id'] = config['username']
```

Comment:
This line gives an error: `ValueError: UserIDs start with @`.
Could you fix this by changing this line to:
```
loginargs['user_id'] = '@{0}:{1}'.format(
    config['username'],
    config['domain'])
```
PEP8 likes a newline after 87 :wink: